src/main/scala/com/snowplowanalytics/s3/loader/processing/Batch.scala

```diff
@@ -1,8 +1,20 @@
+/*
+ * Copyright (c) 2014-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
```

Comment:
Should be in https://github.com/snowplow/snowplow-s3-loader/pull/217/commits/b56ec354a084bf1fd0cd975418f381af17322893